.cilibs/prepare_directories.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+BUILD_OS=linux
+BUILD_ARCH=amd64
+
+echo "Prepare directories"
+sudo mkdir /floodgate
```

Comment:
Maybe you can use here `prepare_extra_directories.sh` script using for example `source` command?

Reply:
Done